kytos/core/napps/manager.py

```diff
@@ -79,8 +79,8 @@ def install(self, napp_uri, enable=True):
             return False
 
         if not napp.repository:
-            # Check here what is configured repo
-            pass
+            options = self._controller.options
+            napp.repository = json.loads(options.napps_repositories)[0]
```

Comment:
I think it would be better to set the options.napps_repositories as an array where the config file is read. After that, you will also have to change a line in manager.py